reliability-engineering/terraform/modules/concourse-web/autoscaling-group.tf

```diff
@@ -1,20 +1,20 @@
 resource "aws_autoscaling_group" "concourse_web" {
   name                = "${var.deployment}-concourse-web"
-  max_size            = "${var.desired_capacity * 2}"
+  max_size            = var.desired_capacity * 2
   min_size            = 0
-  desired_capacity    = "${var.desired_capacity}"
-  vpc_zone_identifier = ["${var.private_subnet_ids}"]
-  target_group_arns   = ["${aws_lb_target_group.concourse_web.arn}"]
-  load_balancers      = ["${aws_elb.concourse_web.name}"]
+  desired_capacity    = var.desired_capacity
+  vpc_zone_identifier = var.private_subnet_ids
+  target_group_arns   = [aws_lb_target_group.concourse_web.arn]
+  load_balancers      = [aws_elb.concourse_web.name]
 
-  launch_template = {
-    id      = "${aws_launch_template.concourse_web.id}"
-    version = "$$Latest"
```

Comment:
Is this change correct?

Reply:
looks like it: https://www.terraform.io/docs/providers/aws/r/autoscaling_group.html#with-latest-version-of-launch-template